cli/cmd/compliance.go

```diff
@@ -435,3 +441,49 @@ func statusToProperTypes(status string) string {
 		return "Unknown"
 	}
 }
+
+func validateRecommendationID(s string) bool {
+	match, _ := regexp.MatchString(RecommendationIDRegex, s)
+	return match
+}
+
+func outputResourcesByRecommendationID(report api.CloudComplianceReport) error {
+	violations := filterResourcesByRecommendationID(report, compCmdState.RecommendationID)
+
+	if len(violations) == 0 {
+		cli.OutputHuman("\nNo resources found affected by '%s'\n", compCmdState.RecommendationID)
```

Comment:
no need of carrier return
```suggestion
		cli.OutputHuman("No resources found affected by '%s'\n", compCmdState.RecommendationID)
```